network/wsNetwork.go

```diff
@@ -1742,6 +1729,41 @@ func (wn *WebsocketNetwork) meshThread() {
 	}
 }
 
+func (wn *WebsocketNetwork) refreshRelayArchivePhonebookAddresses() {
+	// TODO: only do DNS fetch every N seconds? Honor DNS TTL? Trust DNS library we're using to handle caching and TTL?
+	dnsBootstrapArray := wn.config.DNSBootstrapArray(wn.NetworkID)
+
+	// If there is more than one entry in the DNS bootstrap array, we treat the first as 'primary',
+	// second as 'secondary' and attempt to merge/dedup the lists of relay records. Additional entries
+	// will be processed independently.
+	if len(dnsBootstrapArray) >= 2 {
+		primaryRelayAddrs, primaryArchiveAddrs := wn.getDNSAddrs(dnsBootstrapArray[0])
+		secondaryRelayAddrs, secondaryArchiveAddrs := wn.getDNSAddrs(dnsBootstrapArray[1])
+		dedupedRelayAddresses := wn.mergePrimarySecondaryRelayAddressLists(wn.NetworkID, primaryRelayAddrs, secondaryRelayAddrs)
+
+		wn.updatePhonebookAddresses(dedupedRelayAddresses, append(primaryArchiveAddrs, secondaryArchiveAddrs...))
+		//TODO: Sanity check these work in unit tests
+		dnsBootstrapArray = dnsBootstrapArray[2:]
+	}
+
+	for _, dnsBootstrap := range dnsBootstrapArray {
+		relayAddrs, archiveAddrs := wn.getDNSAddrs(dnsBootstrap)
+		wn.updatePhonebookAddresses(relayAddrs, archiveAddrs)
+	}
+}
+
+func (wn *WebsocketNetwork) updatePhonebookAddresses(relayAddrs []string, archiveAddrs []string) {
+	if len(relayAddrs) > 0 {
+		wn.log.Debugf("got %d relay dns addrs, %#v", len(relayAddrs), relayAddrs[:imin(5, len(relayAddrs))])
```

Comment:
Why truncate the log to 5?

Reply:
I think they were avoiding printing out 100s of these (this chunk is migrated from its original home in meshThread)